diff --git a/debian/config/defines b/debian/config/defines
index 1f3be9cc827f..7f07789b61a7 100644
--- a/debian/config/defines
+++ b/debian/config/defines
@@ -112,3 +112,6 @@ gcc-4.9: gcc-4.9 <!stage1 !cross>, gcc-6-@gnu-type-package@:native <!stage1 cros
 # initramfs-generators
 initramfs-fallback: linux-initramfs-tool
 initramfs-tools: initramfs-tools (>= 0.120+deb8u2)
+
+[packages]
+libc-dev: false
